src/LibChorus/VcsDrivers/Mercurial/Revision.cs

```diff
@@ -5,11 +5,11 @@
 
 namespace Chorus.VcsDrivers.Mercurial
 {
-	[Serializable]
+	/// <summary>
+	/// this class is json serialized, so don't change the names of the properties or store service objects in it that can't be serialized
```

Comment:
I wonder if this would make the comment easier to understand:

```suggestion
	/// this class is json serialized, so don't change the names of the properties and don't store service objects in it that can't be serialized
```